score-client/src/main/java/bio/overture/score/client/config/StorageConfig.java

```diff
@@ -18,9 +18,6 @@ public class StorageConfig {
   @Value("${storage.url}")
   private String endpoint;
 
-  @Value("${kf.download.partSize}")
-  private int partSize;
```

Comment:
Do not remove this instance variable. as it is required on the `partCalculator` method

Reply:
As we have removed kf profile from the applicaition.yml,` partsize `was part of kf profile which was causing NullPointerException. So we have added a default value to the partsiz.